simulation/agents.py

```diff
@@ -539,9 +539,9 @@ def install_heating_system(
 
         if self.boiler_upgrade_grant_available:
             if heating_system == HeatingSystem.HEAT_PUMP_AIR_SOURCE:
-                self.boiler_upgrade_grant_used = 5_000
+                self.boiler_upgrade_grant_used = 7_500
```

Comment:
Non-blocking: This constant is reused in multiple places. Should this go in `constants.py` as well?